.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,7 +33,7 @@ implementation proposal in the referenced issues.
 <!-- Make sure that this pull request is complete. -->
 
 - [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
-- [ ] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
+- [ ] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`. If yes, the target branch is set to `master`, otherwise the target branch is set to `develop`.
```

Comment:
This implies that "if no", we can merge anything to `develop` while we also have our compatibility commitment there, although to a lesser extent.

I would reword this to:

```
- [ ] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
```